wandb/sdk/internal/sender.py

```diff
@@ -285,6 +290,10 @@ def send_exit(self, data):
         exit = data.exit
         self._exit_code = exit.exit_code
         logger.info("handling exit code: %s", exit.exit_code)
+        runtime = exit.runtime
+        logger.info("handling runtime: %s", exit.runtime)
+        self._metadata_summary["_wandb"]["runtime"] = runtime
```

Comment:
I think this would be cleaner as all the keys in _wandb. so a single layer dict instead of nested.

Reply:
didn't we want another dictionary for all the `_wandb` entries?

Reply:
yes.... but if it is just for wandb entries, then should it be:
```
self._metadata_summary_wandb["runtime"] = runtime
```

Reply:
oh i see what you mean. sure i'll fix it